src/website/assets/styles/_docs-header.scss

```diff
@@ -7,6 +7,7 @@
 
     @include respond-to(lg) {
       position: fixed;
+      z-index: 2006;
```

Comment:
Could be problematic for non-mobile scenarios as it hides the drawers close button. Might be safer to remove for now.